.github/ISSUE_TEMPLATE/general-issue-template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: General issue template
+about: Issue on the EPUB 3.3 specifications or related Working Group Notes
+title: ''
+labels: ''
+assignees: ''
+
+---
+This repository is only for issues with the following W3C documents. Questions or issues with specific reading systems, achieving specific styling, understanding validation errors, and similar will be considered as out of scope and will be closed without further action.
+
+* [EPUB 3 Overview](https://www.w3.org/TR/epub-overview-33/), WG Note
+* [EPUB 3.3](https://www.w3.org/TR/epub-33/), Recommendation
+* [EPUB Reading Systems 3.3](https://www.w3.org/TR/epub-rs-33/), Recommendation
+* [EPUB 3 Multiple-Rendition Publications 1.1](https://www.w3.org/TR/epub-multi-rend-11/), WG Note
+* [EPUB 3 Text-to-Speech Enhancements 1.0](https://www.w3.org/TR/epub-tts-10/), WG Note
+* [EPUB 3 Structural Semantics Vocabulary 1.1](https://www.w3.org/TR/epub-ssv-11/), WG Note
+* [EPUB Accessibility 1.1](https://www.w3.org/TR/epub-a11y-11/), Recommendation
+* [EPUB Accessibility Techniques 1.1](https://www.w3.org/TR/epub-a11y-tech-11/), WG Note
+* [EPUB Accessibility - EU Accessibility Act Mapping](https://www.w3.org/TR/epub-a11y-eaa-mapping/), WG Note
+* [The EPUB Accessibility exemption property](https://www.w3.org/TR/epub-a11y-exemption/), WG Note
+* [EPUB Type to ARIA Role Authoring Guide 1.1](https://www.w3.org/TR/epub-aria-authoring-11/), WG Note
+* [EPUB Fixed Layout Accessibility](https://www.w3.org/TR/epub-fxl-a11y/), WG Note
+
+
+Some possible alternative locations get help to your problems:
+
+* the [epub forum](https://www.mobileread.com/forums/forumdisplay.php?f=179) at mobileread
+* the [W3C publishing community group](https://github.com/w3c/publishingcg/issues)
+* the [ebooks section](https://ebooks.stackexchange.com/questions/tagged/epub) at stackexchange
+
```

Comment:
It might be useful to add starter questions.
```suggestion
## Which standard or note is this issue related to?

Please include the title and version number. For bug reports, include the relevant section.

## Describe the problem? 

What is not working? What would you like to see improved? For bug reports, cite the relevant text from the specification.

## Describe the fix or new feature you propose?

```